index.html

```diff
@@ -5,8 +5,60 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+    <script src="src/index.js"></script> 
+    <!-- <script src="./node_modules/axios/dist/axios.min.js"></script> -->
 </head>
 <body>
-    
+    <header>
+        <h1 id="Weather-Report"> Weather Report </h1>
+        <h2 id="For-the-lovely-city-of"> For the lovely city of </h2>
+        <h3 id="City-Name-Top"> ✨ Seattle ✨ </h3>
+    </header>
+        <div class="container">
+            <div class="grid-1">
+                    <div id="Temperature"> 
```

Comment:
Here and throughout your code try to use more semantic HTML elements. What tag do you think might be more semantic here?